groclient/lib.py

```diff
@@ -150,6 +150,16 @@ def get_version_info():
     return _VERSIONS.copy()
 
 
+def compute_value(value, from_convert_factor, to_convert_factor):
```

Comment:
minor - might be better to call it `convert_value` , "compute" is little unclear what's the calculation is doing

Reply:
this is a helper function which deals with the calculation, while client.convert_unit focuses on data point level.

Reply:
renamed